### DISCUSSION TITLE
останавливается загрузка ролика youtube на 19 секунде

### DISCUSSION THREAD

**Discussion by hhate5** [bol-van/zapret#981] [Q&A]
Периодически наблюдаю остановку проигрывания/загрузки ролика на youtube именно на 19 секунде, если обновить страницу то ролик загрузится и проиграется нормально по всему хронометражу. Проявляется не постоянно а с разной периодичностью. Примерно 1-2 ролика из 10 открытых. Такое же поведение на телевизоре.
Браузер хром версии zapret 69.8, все настроено в ethware на роутере keenetic
настройки в барузере quic и tls в default, zapret работает по хостлисту.
В этот момент проблемы наблюдаю такой вывод в отладочную консоль chrome:
![image](https://github.com/user-attachments/assets/d37739dc-d0e7-4e8e-ba22-d2c636cc262b)
DNS doh comss.

**Comment by GuFFy12** [comment]
Та же проблема, те же ошибки. Затрагивает все версии ютуба (мобильная, пк), браузеры (safari, chrome), устройства с запущенным zapret (keenetic, windows). Пробовал менять стратегии, безрезультатно. Предполагаю что вина не на стороне обхода блокировок, а на стороне dpi / google cache серверов. Причем проблема не решается уже месяца 2. Отпишу в сабж дополнительно после того как попробую мобильный интернет, может будут изменения.

**Reply by Proxxxy** [reply]
Проверял на всех видах интернета, исход такой же.

**Reply by GuFFy12** [reply]
Да, на мобильном интернете та же шляпа :(

**Reply by GuFFy12** [reply]
а да, чет тупанул, обновил конфиг с новой версии:
```sh
NFQWS_OPT="
--filter-tcp=80,443 --dpi-desync=fake,multidisorder --dpi-desync-split-pos=1,midsld --dpi-desync-fooling=md5sig,badsum --dpi-desync-fake-tls=/opt/zapret/files/fake/tls_clienthello_www_google_com.bin <HOSTLIST> --new
--filter-udp=443 --dpi-desync=fake --dpi-desync-repeats=6 --dpi-desync-fake-quic=/opt/zapret/files/fake/quic_initial_www_google_com.bin <HOSTLIST>
"
```
теперь все работает)

**Reply by hhate5** [reply]
Проверил работу на этих опциях ничего не поменялось, зависания на 19 секунде так же присутствуют.

**Reply by brnsky** [reply]
> а да, чет тупанул, обновил конфиг с новой версии:
> 
> ```shell
> NFQWS_OPT="
> --filter-tcp=80,443 --dpi-desync=fake,multidisorder --dpi-desync-split-pos=1,midsld --dpi-desync-fooling=md5sig,badsum --dpi-desync-fake-tls=/opt/zapret/files/fake/tls_clienthello_www_google_com.bin <HOSTLIST> --new
> --filter-udp=443 --dpi-desync=fake --dpi-desync-repeats=6 --dpi-desync-fake-quic=/opt/zapret/files/fake/quic_initial_www_google_com.bin <HOSTLIST>
> "
> ```
> 
> теперь все работает)

так это конкретно настройка для вас, а не для всех, у меня, например, только через tpws работает. у меня на днях тоже такая же issue с 19 секундами появилась вчера

**Reply by brnsky** [reply]
> Проверил работу на этих опциях ничего не поменялось, зависания на 19 секунде так же присутствуют.

у вас не получилось пофиксить проблему?

**Reply by hhate5** [reply]
Играл с опциями, но найти решение не удалось, пока приходится мириться с перезагрузкой при зависаниях на 19 секунде

**Reply by brnsky** [reply]
> Играл с опциями, но найти решение не удалось, пока приходится мириться с перезагрузкой при зависаниях на 19 секунде

 только что попробовал с куиком конфиг того парня и вырубил QUIC в двух браузерах (firefox, chrome) и все заработало

**Reply by eror452lol** [reply]
> > Играл с опциями, но найти решение не удалось, пока приходится мириться с перезагрузкой при зависаниях на 19 секунде
> 
> только что попробовал с куиком конфиг того парня и вырубил QUIC в двух браузерах (firefox, chrome) и все заработало

подскажите пожалуйста, куда вставлять какие опции

**Comment by brnsky** [comment]
> Периодически наблюдаю остановку проигрывания/загрузки ролика на youtube именно на 19 секунде, если обновить страницу то ролик загрузится и проиграется нормально по всему хронометражу. Проявляется не постоянно а с разной периодичностью. Примерно 1-2 ролика из 10 открытых. Такое же поведение на телевизоре.
> Браузер хром версии zapret 69.8, все настроено в ethware на роутере keenetic
> настройки в барузере quic и tls в default, zapret работает по хостлисту.
> В этот момент проблемы наблюдаю такой вывод в отладочную консоль chrome:
> ![image](https://github.com/user-attachments/assets/d37739dc-d0e7-4e8e-ba22-d2c636cc262b)
> DNS doh comss.

Кстати, такая же проблема с использованием comss DNS + zapret началась. До этого не было, только теперь сайт Ютуба вообще грузится очень долго либо бесконечно. У вас удалось починить ?

**Comment by Dali1896** [comment]
Заметил, что на тех роликах в которых произошло зависание в инструментах браузера какие-то ошибки по адресам
с доменом doubleclick.net.  Добавил в файл /opt/zapret/ipset/zapret-hosts-google.txt строки 
 doubleclick.net
googleads.g.doubleclick.net
Проблема ушла, иногда происходит подвисание, но затем продолжается проигрывание, не требуется перезагрузки страницы.

**Comment by cvnkk** [comment]
у меня лично проблема пропала после того как я включил quic протокол в хроме.

**Reply by shinikakeru** [reply]
люблю целую тебя

**Comment by DimetryBadcoder** [comment]
Я эту проблему решал, изменив стратегию, но я совершенно не помню, в каком параметре была проблема

**Comment by usernamePaper** [comment]
мне помогло отключение блокировщика рекламы
